examples/aks_service/scripts/store_kube_token.py

```diff
@@ -5,4 +5,7 @@
 
 
 client = get_rest_client()
-client.secrets.create('kubernetes_token',  base64.b64decode(inputs['kube_token']), update_if_exists=True)
+client.secrets.create(
+    'kubernetes_token',
+    base64.b64decode(inputs['kube_token']),
```

Comment:
`base64.b64decode` returns `bytes` while `SecretClient.create()` expects `unicode`.

I guess it's one of those places where you should “wrap” external library with `.decode('utf-8')`:

```
client.secrets.create(
    'kubernetes_token',
    base64.b64decode(inputs['kube_token']).decode('utf-8'),
    update_if_exists=True)
```

Reply:
yeah it's super stupid that they chose bytes for base64, we talked about this a couple times :/